src/views/Submit.vue

```diff
@@ -665,15 +667,31 @@ export default {
 			this.loading = true
 
 			try {
-				await axios.post(
-					generateOcsUrl('apps/forms/api/v3/forms/{id}/submissions', {
-						id: this.form.id,
-					}),
-					{
-						answers: this.answers,
-						shareHash: this.shareHash,
-					},
-				)
+				if (this.submissionId) {
+					await axios.post(
```

Comment:
See comment for ApiController
```suggestion
					await axios.put(
```